File: samples/Objective-C/fizzbuzz.m
// FizzBuzz in Objective-C
#import <stdio.h>

main() {
	for (int i=1; i<=100; i++) {
		if (i % 15 == 0) {
		    printf("FizzBuzz\n");
		} else if (i % 3 == 0) {
		    printf("Fizz\n");
		} else if (i % 5 == 0) {
		    printf("Buzz\n");
		} else {
		    printf("%i\n", i);
		}
	}
}

File: samples/OCaml/fizzbuzz.ml
let output x =
  match x mod 3 = 0, x mod 5 = 0 with
    true,  true  -> "FizzBuzz"
  | true,  false -> "Fizz"
  | false, true  -> "Buzz"
  | false, false -> string_of_int x

let _ =
  for i = 1 to 100 do print_endline (output i) done
